app/controllers/admin/schools/cohorts_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Admin
+  class Schools::CohortsController < Admin::BaseController
+    skip_after_action :verify_authorized
+    before_action :set_school
+
+    def index
+      @school_cohorts = policy_scope(SchoolCohort)
+        .joins(:school)
+        .where(school: @school)
+        .joins(:cohort)
+        .order("cohorts.start_year ASC")
+    end
```

Comment:
Do we need to paginate it? There will be thousands of schools with a school cohort :/

Reply:
I guess the second part is: Do we need this list to be searchable? Not necessarily in this PR, just whether it's taken into account in general.

Reply:
This is only the cohorts for a single school?